solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -1,84 +1,102 @@
 # Setup K8s Cluster and Deploy CORTX Stack
 
-   Please find below steps to setup K8s cluster and deploy CORTX Stack.
+The following sections discusses how to set up the K8s cluster and deploy the CORTX Stack. The minimum prerequisites are enlisted to make sure the cluster set up process is smooth.
 
 ## Prerequisite 
-### Minimum specification required for Cortx Stack
+Following are the prerequisites to set up the K8s cluster and deploy the CORTX stack
+
+### Minimum system specifications required for Cortx Stack
    - RAM: 16GB
    - CPU: 8Core
    - DISK: Need 8 Disk with 25GB per disk
 
-### There are two types config for `solution.yaml` file manual and automated. If you have automated config type for `solution.yaml`, please make sure your vm having below drives available
+### Types of configurations available for `solution.yaml` file
```

Comment:
This is not a prerequisite. This should go into a separate section in Deploy the Cortx Stack on K8s Cluster

Reply:
@swatid-seagate Could you please address this.